src/application.ts

```diff
@@ -16,6 +16,9 @@ export class HelloWorldApp extends Application {
     super({
       components: [AuthenticationComponent],
       sequence: MySequence,
+      http: {
+        port: 0,
+      },
```

Comment:
Changed it to use ephemeral port for test server only.  User 3000 otherwise.
Avoid a duplicate server start in the test mode.